pdns/dnsdist-tcp.cc

```diff
@@ -715,7 +716,7 @@ void* tcpAcceptorThread(void* p)
         continue;
       }
 #endif
-
+      SSetsockopt(ci->fd, SOL_TCP, TCP_NODELAY, 1); // disable NAGLE
```

Comment:
Just a nit but we have a nice `setTCPNoDelay()` function in `misc.hh` :-) It also uses `IPPROTO_TCP` instead of `SOL_TCP`, which is more portable.